apps/proto/lib/lexical/proto/enum.ex

```diff
@@ -25,6 +45,18 @@ defmodule Lexical.Proto.Enum do
     end
   end
 
+  defp determine_type(i) when is_integer(i) do
```

Comment:
Now that Typespec has a `literal_type` function, do we want to make that public and use it here?

Reply:
Typespec's function is too broad; Enums can only have string or integer values. Everything else is an error.

Reply:
Ah got it